src/os/filestore/HashIndex.cc

```diff
@@ -396,7 +396,7 @@ int HashIndex::read_settings() {
   if (r == -ENODATA)
     return 0;
   if (r < 0) {
-    derr << __func__ << " error reading settings: " << cpp_strerror(r) << dendl;
+    dout(0) << __func__ << " error reading settings: " << cpp_strerror(r) << dendl;
```

Comment:
@tchaikov 
Because the program merrily continues with the default properties.
So it is no more than perhaps a strong warning